tools/Dockerfile

```
@@ -10,9 +10,8 @@ RUN microdnf update && \
     microdnf clean all
 
 # The bcrypt tool is needed by NiFi to locally encrypt the admin password that is mounted as a secret in cleartext
-RUN curl -L https://github.com/bitnami/bcrypt-cli/releases/download/v1.0.2/bcrypt-linux-amd64.tar.gz | tar -xzC . && \
-    mv bcrypt-linux-amd64 /bin/bcrypt && \
-    chmod +x /bin/bcrypt
+COPY tools/bin/stackable-bcrypt-1.0-SNAPSHOT-jar-with-dependencies.jar /bin
+RUN mv /bin/stackable-bcrypt-1.0-SNAPSHOT-jar-with-dependencies.jar /bin/stackable-bcrypt.jar
```

```suggestion
COPY tools/bin/stackable-bcrypt-1.0-SNAPSHOT-jar-with-dependencies.jar /bin/stackable-bcrypt.jar
```

We might also want to rename the jar to something like nifi-bcrypt?